_episodes/08-kokkos-gpu.md

```diff
@@ -100,16 +98,24 @@ In the following discussion, we'll work on a few exercises to get familiarized o
 these aspects to some extent.
 
 > ## Exercise: Performance penalty due to use of mixed styles
->   1. First, let us take the input for the LJ-system from the Exercise 1 of the GPU
->      section in Episode 5 (***ADD REF***, let us call this as *Input1*). Run this input
->      using all
->      the visible devices in a node available to you and use **Kokkos**/GPU as the
->      accelerator package using the following settings ***THESE ARE SYSTEM SPECIFIC***:
->      `-k on g 4 -sf kk -pk kokkos newton off neigh full comm device cuda/aware off`. Use
->      the number of MPI tasks that equals to the number of devices.
->   2. Measure the performance of this run in units of `timesteps/s`.
->   3. Now, modify the above LJ-input file and append the following lines to the end of
->      the file:
+>
+> 1. First, let us take the input for the LJ-system from
+>    [episode 5]({{page.root}}/05-accelerating-lammps/#learn-to-call-the-gpu-package-from-the-command-line).
+>    Run this input using all the visible devices in a node available to you and use **Kokkos**/GPU
+>    as the accelerator package using the following settings;
+>    **CHECK ME!**
```

Comment:
@prithwish-ichec, please check list below. Is it presented in the correct way/notation?